dcp_prototype/backend/browser/code/common/db_utils.py

```diff
@@ -93,7 +139,21 @@ def query_file(self, file_id):
         :param file_id: File ID
         :return: File query result
         """
-        return self._get(File, file_id)
+        file = self._get(File, file_id)
```

Comment:
Why do you need to pass in File when you are importing it at the top?

Reply:
The first parameter of `_get` specifies which table to query